packages/java/parser-jvm-plugin-backbone/src/main/java/dev/hilla/parser/plugins/backbone/MethodPlugin.java

```diff
@@ -80,4 +99,10 @@ private ApiResponses createResponses() {
         var response = new ApiResponse().description("");
         return new ApiResponses().addApiResponse("200", response);
     }
+
+    private Optional<Node<?, ?>> findClosestEndpoint(NodePath<?> nodePath) {
+        return nodePath.getParentPath().stream()
+                .<Node<?, ?>> map(NodePath::getNode)
+                .filter(n -> n instanceof EndpointNode).findFirst();
```

Comment:
I'd keep Anton's syntax as it is not a dynamic check.

Reply:
`isInstance` is mainly used when the class to check is dynamic. `instanceof` is the right syntax here, although yes, you'll get the same result with both implementations. You'll never be able to measure any performance gain here, and I not sure it exists, because calling a dynamic check could prove more expensive than creating a lambda.

Reply:
Yeah, I had the same concerns about it. However, there is an [answer](https://stackoverflow.com/questions/36435670/whats-faster-instanceof-or-isinstance) from StackOverflow that says that they are compiled to the same bytecode sequence actually.

Reply:
Well, yeah, that's also true, it's more about own preferences. I just used that approach across the code (you could find a lot of `SomeClass.class::cast` method usage), so it could be another argument of mine to use a method ref here. But I don't really mind which one to choose.